src/govuk/components/cookie-banner/template.njk

```diff
@@ -0,0 +1,57 @@
+{% from "../button/macro.njk" import govukButton %}
+
+<div class="govuk-cookie-banner" role="region" aria-label="{{ params.ariaLabel | default("Cookie banner") }}">
+  {% for banner in params.banners %}
+    {%- set classNames = "govuk-cookie-banner__container govuk-width-container" -%}
+    {% if banner.classes %}
+      {%- set classNames = classNames + " " + banner.classes -%}
+    {% endif -%}
+
+    <div class="{{classNames}}" {% if banner.role %}role="{{banner.role}}"{% endif %}
```

Comment:
Minor, but it'd be good to use Nunjucks' [whitespace control](https://mozilla.github.io/nunjucks/templating.html#whitespace-control) to avoid outputting extra spaces where they're not needed. They can be a bit confusing to get the hang of – give me a shout if you want to pair on it or anything.